web/src/features/time/TimeControllerWrapper.tsx

```diff
@@ -32,7 +32,7 @@ function BottomSheetWrappedTimeController() {
       blocking={false}
       header={<TimeHeader />}
     >
-      <TimeController className="p-2 pt-1 min-[370px]:px-4" />
+      <TimeController className="p-2 min-[370px]:px-4" />
```

Comment:
On smaller screens (and languages with longer text) it would be nice to have some top padding again:

**Before:**


![Screenshot 2024-08-30 at 14 55 23](https://github.com/user-attachments/assets/704d44bd-2d20-472e-b586-626a8e14e47b)


**This PR:**

![Screenshot 2024-08-30 at 14 55 17](https://github.com/user-attachments/assets/f0b29631-926d-498d-a230-046462a73b28)

Reply:
Good catch Mads! 🙌 Some extra padding around the header would be nice 🫶

Reply:
The padding itself is actually doubled but I think it's the overflow that's the problem by setting a fixed hight of the "title" to avoid CLS caused by the badge.

But I'll try out a few solutions to see what is working best 🙂